docker-haste-server.md

```diff
@@ -0,0 +1,159 @@
+# [linuxserver/haste-server](https://github.com/linuxserver/docker-haste-server)
```

Comment:
This file should be removed.

Reply:
or remove README.md and rename this accordingly